Final_Project/Sender.cpp

```diff
@@ -20,28 +20,28 @@
 --	NOTES:
 --	Pass this thread function to Sender thread to send out a frame
 --------------------------------------------------------------------------------------*/
-DWORD WINAPI WriteToPort(LPVOID hComm, LPVOID frameToSend)
-{
-	DWORD dwWrite = NULL;
-	DWORD dwBytesWritten = 0;
-
-	DWORD dwEvent;
-	// commMask should be combined with Receiver and set in Main
-	SetCommMask(hComm, EV_CTS);
-
-	while (curState == "SEND")
-	{
-		if (WaitCommEvent(hComm, &dwEvent, NULL))
-		{
-			if(dwEvent & EV_CTS)
-			{
-				do
-				{
-					WriteFile(hComm, frameToSend, 1024, &dwBytesWritten, NULL);
-				} while (isClearToSend);
-			}
-		}
-	}
-
-	return 0;
-}
+//DWORD WINAPI WriteToPort(LPVOID portHandle)
```

Comment:
Is this function meant to just stay commented out?

Reply:
No i need to refactor this to reflect our layered approach.
Will do